src/devtools/client/debugger/src/components/A11yIntention.js

```diff
@@ -3,28 +3,22 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 //
-import React from "react";
+import React, { useState } from "react";
```

Comment:
:hand: While we're at it, bonus request: can you convert this file to TypeScript?

Looking at it, I'm pretty sure that _all_ you need to do is:

- Rename it to `.tsx`
- Add a type declaration for the props that includes `children` (probably `{children: React.ReactNode}`), and use that

Reply:
Roger that!